apps/ledger-live-mobile/src/families/osmosis/operationDetails.js

```diff
@@ -0,0 +1,272 @@
+// @flow
+import React, { useCallback } from "react";
+import { Linking } from "react-native";
+import { useTranslation } from "react-i18next";
+import {
+  getDefaultExplorerView,
+  getAddressExplorer,
+} from "@ledgerhq/live-common/explorers";
+import type { Account, OperationType, Operation } from "@ledgerhq/types-live";
+import { useCosmosFamilyPreloadData } from "@ledgerhq/live-common/families/cosmos/react";
+import { formatCurrencyUnit } from "@ledgerhq/live-common/currencies/formatCurrencyUnit";
+import { BigNumber } from "bignumber.js";
+import { getAccountUnit } from "@ledgerhq/live-common/account/helpers";
+
+import type { CosmosDelegationInfo } from "@ledgerhq/live-common/families/cosmos/types";
+import { useSelector } from "react-redux";
+import Section from "../../screens/OperationDetails/Section";
+import { urls } from "../../config/urls";
+import { discreetModeSelector, localeSelector } from "../../reducers/settings";
+
+function getURLFeesInfo(op: Operation): ?string {
+  return op.fee.gt(200000) ? urls.cosmosStakingRewards : undefined;
```

Comment:
Maybe:

```suggestion
  return op.fee.gt(200000) ? urls.osmosisStakingRewards : undefined;
```

Reply:
Nice catch! I thought I had checked these over but clearly not closely enough.